script.js

```diff
@@ -0,0 +1,42 @@
+const draggableList = document.getElementById("draggable-list");
+const check = document.getElementById("check");
+
+const richestPeople = [
+  "Jeff Bezos",
+  "Bill Gates",
+  "Warren Buffett",
+  "Bernard Arnault",
+  "Carlos Slim Helu",
+  "Amancio Ortega",
+  "Larry Ellison",
+  "Mark Zuckerberg",
+  "Michael Bloomberg",
+  "Larry Page",
+];
+
+const listItems = [];
+
+let dragStartIndex;
+
+function createList() {
+  [...richestPeople]
+    .map((a) => ({ value: a, sort: Math.random() }))
+    .sort((a, b) => a.sort - b.sort)
+    .map((a) => a.value)
+    .forEach((person, index) => {
+      const listItem = document.createElement("li");
+      listItem.setAttribute("data-index", index);
+
+      listItem.innerHTML = `
+
+      <span class="number">${index + 1}</span>
+      <div class="draggable">
+        <p class="person-name">${person}</p>
+        <i class="fas fa-grip-lines"></i>
+      </div>
+    `;
+      listItems.push(listItem);
+      draggableList.appendChild(listItem);
+    });
+}
+createList();
```

Comment:
Your Javascript code those not work! kindly check because I can not drag or check the order by clicking on the `check order` button.